lib/main_common.pm

```diff
@@ -534,8 +534,28 @@ sub load_system_role_tests {
         loadtest "installation/logpackages";
     }
     loadtest "installation/disable_online_repos" if get_var('DISABLE_ONLINE_REPOS') && !get_var('OFFLINE_SUT');
-    loadtest "installation/installer_desktopselection" if is_opensuse;
-    loadtest "installation/system_role" if is_caasp('kubic');
+    if (is_opensuse) {
+        if (is_leap('<=15.0')) {
```

Comment:
These both if should be combineable now, just `is_leap('<=15.0')` without the `is_opensuse` part should be enough